app/views/transfers/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= render 'shared/navbar', group: @groups %>
+<button class="back"><%= link_to "back", groups_path  %></button>
+
+
+<div class="group_main_div">
+  <h3 class="group_main_title">Expenses</h3>
+  <% if @transfers.empty? %>
+    <h3 class="group_main_title">You do not have any transfers yet</h3>
+  <% else %>
+    <div>
+      <div class="transact_div">
+      <h1><%= @group.name %></h1>
+      <span>$<%= @transfers.sum(:amount) %></span>
+      </div>
+      
+        <% @transfers.each do |transfer| %>
+        <div class="transact_subdiv">
+              <h2 class="name"><%= transfer.name %></h2>
+              <p class= "dollar">$<%= transfer.amount %></p>
+      </div>
+         
+        <% end %>
+    
+    </div>
+  <% end %>
+  
+   <button class='category' ><%= link_to 'Add new transaction', new_group_transfer_path %></button>
+</div>
```

Comment:
- [ ] Dear @lawrahkonwea, when I try to add a transaction without choosing any of the groups, The app runs into a problem, what is expected is to make the checkbox a required field and prevent the user from creating a transaction without choosing at least one group. 
### Errore while creating a transaction without choosing a group. 
 
![image](https://github.com/lawrahkonwea/Budget_APP/assets/69077061/462e2da6-4c05-45fb-9527-2dff80cd4d46)